templates/angular/src/contract-name.js

```diff
@@ -0,0 +1 @@
+module.exports = ''
```

Comment:
@faragly does this file need to be included in the source? It seems like a build artifact that should be gitignored, since it will change based on whether you're using `yarn dev` or `yarn start`, and could also change between different runs of `yarn dev` if you clear your `neardev/dev-account` file

Reply:
See https://github.com/near/create-near-app/pull/510

Reply:
You are right, this file changes frequently and should be gitignored. But if we delete this file, then an error occurs when importing a non-existent file (these errors are now visible in Travis). I have not yet found a concise way to get rid of this file.